subprojects/core-api/src/main/java/org/gradle/api/initialization/Settings.java

```diff
@@ -208,6 +211,24 @@
      */
     StartParameter getStartParameter();
 
+    /**
+     * Creates a {@code Provider} implementation based on the provided value.
+     *
+     * @param value The {@code java.util.concurrent.Callable} use to calculate the value.
+     * @return The provider. Never returns null.
+     * @throws org.gradle.api.InvalidUserDataException If the provided value is null.
+     * @see org.gradle.api.provider.ProviderFactory#provider(Callable)
+     * @since 6.8
+     */
+    <T> Provider<T> provider(Callable<T> value);
```

Comment:
:x: Let's not add this method to `Settings` as part of this change. The same can be done using `providers.provider {}`.

Reply:
I was trying to mimic the API used in `Project` - is that not the ideal API?  (For the record, I agree, the helper method isn't necessary)

Reply:
Done.